test/parallel_testsuite.py

```diff
@@ -192,6 +218,30 @@ def updateResult(self, result):
     self.buffered_result.updateResult(result)
     result.stopTest(self.test)
     result.core_time += self.test_duration
+    self.log_test_run_for_visualization()
+
+  def log_test_run_for_visualization(self):
+    if os.getenv('EMTEST_VISUALIZE') and (self.test_result != 'skipped' or self.test_duration > 0.2):
+      profiler_logs_path = os.path.join(tempfile.gettempdir(), 'emscripten_toolchain_profiler_logs')
+      os.makedirs(profiler_logs_path, exist_ok=True)
+      profiler_log_file = os.path.join(profiler_logs_path, 'toolchain_profiler.pid_0.json')
+      prof = open(profiler_log_file, 'a')
```

Comment:
`with prof as open(profiler_log_file, 'a'):`

Reply:
Looks like it can move down to line 243 too

Reply:
check